src/families/cosmos/types.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+
+import type { BigNumber } from "bignumber.js";
+import type {
+  TransactionCommon,
+  TransactionCommonRaw
+} from "../../types/transaction";
+
+export type CoreStatics = {};
+export type CoreAccountSpecifics = {};
+export type CoreOperationSpecifics = {};
+export type CoreCurrencySpecifics = {};
+
+export type CosmosDelegation = {|
+  validatorAddress: string,
+  amount: BigNumber,
+  pendingRewards: BigNumber,
+  isActive: boolean, // TBD is there more status case? if so, switch to status field with an enum string.
```

Comment:
A Delegation is always "active" ; if `isActive` means "generatesRewards" here (what I suspect), then `isActive` is only depending on the validator state.

Technically there are 3 statuses 
- "bonded" : in the active set that generates rewards
- "unbonding" : doesn't generate rewards
- "unbonded" : doesn't generate rewards.

I'm okay with the boolean here, we can discuss it further on other channels

Reply:
Ok! let's aim for something that is closer to technical reality then.

Reply:
For the record :
- unbonding means the validator has been removed from the active set, but has its voting power "frozen" in case they misbehaved (just like a delegator undelegating). This last 21 days
- unbonded means the validator has been removed from the active set for more than 21 days basically